tools/pre_post_processing/steps/vision.py

```diff
@@ -118,6 +118,11 @@ def _create_graph_for_step(self, graph: onnx.GraphProto):
                                 [0.500, -0.587 / 1.402, -0.114 / 1.402]],
                                dtype=np.float32)  # fmt: skip
 
+        # https://github.com/python-pillow/Pillow/blob/63f64de8b5cc6d065f19cc73c865633ab4940a52/src/libImaging/ConvertYCbCr.c#L17
+        # rgb_weights = np.array([[0.299, 0.587, 0.114],
```

Comment:
the commented out values seem to be close to the values above. e.g., `-0.299 / 1.772` vs `-0.16874`. why are they listed here? maybe add some more context in the comment?

Reply:
Was temporary whilst trying to get to the bottom of the diffs. Removed.